code/modules/client/preferences_savefile.dm

```diff
@@ -69,6 +69,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		S.dir.Remove("overhead_chat")
 	if(current_version < 35)
 		see_balloon_alerts = BALLOON_ALERT_ALWAYS
+	if(current_version < 36)
+		WRITE_FILE(S["helmet_style"], HELMET_DEFAULT)
```

Comment:
Current version savefile updates is for variables where the default value has changed / the existing value is outdated, any new values added will be set to the default anyway.